packages/js/product-editor/src/icons/gift.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { createElement } from '@wordpress/element';
+
+const gift = (
+	<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
we should use core components here. You can take a look at the [ShoppingBags](https://github.com/woocommerce/woocommerce/blob/7fe00de7278cbc0d7806e1733b8df7320feef933/packages/js/product-editor/src/images/shopping-bags/index.tsx) image.